test/dom.js

```diff
@@ -270,6 +272,18 @@ describe('dom', function () {
       expect(fromArray(container.children)).to.eql([one, two, three])
     })
   })
+
+  describe('insertChild', function () {
+    it('should insertChild an element and return a restore function', function () {
+      const dummy = document.createElement('div')
+      dummy.id = 'dummy'
+      const restore = insertChild(two, dummy)
+      expect(fromArray(container.children)).to.eql([one, two, three])
+      expect(fromArray(two.children)).to.eql([dummy])
+      restore()
+      expect(fromArray(container.children)).to.eql([one, two, three])
```

Comment:
should we check here if `two.children` has been removed?

Reply:
sure